index.bs

```diff
@@ -2519,7 +2550,23 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
         :   If the user exercises a user agent user-interface option to cancel the process,
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
-            and [=set/remove=] |authenticator| from |issuedRequests|. Throw a "{{NotAllowedError}}" {{DOMException}}.
+            and [=set/remove=] |authenticator| from |issuedRequests|.
+            
+            If the user agent is informing the user of an inability to continue the ceremony
+            due to missing {{AuthenticatorTransport/hybrid}} prerequisites,
+            throw a "{{HybridPrerequisitesError}}" {{DOMException}}.
+
+            If the user agent is prompting the user to complete the ceremony using an authenticator
+            that may be available over the {{AuthenticatorTransport/hybrid}} transport,
+            throw a "{{UserHybridCancellationError}}" {{DOMException}}.
+
+            If the user agent is informing the user that
```

Comment:
nit: does the user agent typically inform the user of any of this?

Reply:
Chrome can when it's handling hybrid `HybridPrerequisitesError`:

![image](https://github.com/user-attachments/assets/66e69967-4b7d-426d-ae76-80600858a4c4)

![image](https://github.com/user-attachments/assets/b93e3997-a984-4b78-9510-020a82634208)

The macOS platform authenticator (obviously not a user agent, to another comment you left earlier) also shows something similar:

![image](https://github.com/user-attachments/assets/fa6bbf7e-cf83-4f20-a8c2-b13b7d03c411)

The fact that we now have platforms able to handle hybrid is complicating these new hybrid errors...

Reply:
Right, but in most cases, it is not the user agent. Chrome on Mac and Chrome on Linux are the exceptions.

Reply:
I'm taking `HybridPrerequisitesError ` out of the running for now, I don't want it to block some of the other errors.